workitem_whitebox_test.go

```diff
@@ -20,15 +22,25 @@ var DB *gorm.DB
 var rwiScheduler *remoteworkitem.Scheduler
 
 func TestMain(m *testing.M) {
+	var err error
+
+	if err = configuration.SetupConfiguration(configuration.DefaultConfigFilePath); err != nil {
+		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	}
+
 	if _, c := os.LookupEnv(resource.Database); c != false {
```

Comment:
But I do need to copy. Simply because the resource.Require needs to take a `testing.T` argument that I have not at hand in this place. Also, the Require function skips a test which is not what I want to do here. This code has not been touched by this change, so lets not discuss it here, shall we?